contributors/design-proposals/aggregated-api-servers.md

```diff
@@ -0,0 +1,266 @@
+# Aggregated API Servers
+
+## Abstract
+
+We want to divide the single monolithic API server into multiple aggregated
+servers. Anyone should be able to write their own aggregated API server to expose APIs they want.
+Cluster admins should be able to expose new APIs at runtime by bringing up new
+aggregated servers.
+
+## Motivation
+
+* Extensibility: We want to allow community members to write their own API
+  servers to expose APIs they want. Cluster admins should be able to use these
+  servers without having to require any change in the core kubernetes
+  repository.
+* Unblock new APIs from core kubernetes team review: A lot of new API proposals
+  are currently blocked on review from the core kubernetes team. By allowing
+  developers to expose their APIs as a separate server and enabling the cluster
+  admin to use it without any change to the core kubernetes repository, we
+  unblock these APIs.
+* Place for staging experimental APIs: New APIs can remain in separate
+  aggregated servers until they become stable, at which point, they can be moved
```

Comment:
I'm not a fan of moving APIs.  I'd rather see us bless some APIs (maybe a "recommended addons" list or something?), but I don't think we should be in the business of taking people's APIs and of growing our surface area further.